lib/Zonemaster/CLI.pm

```diff
@@ -529,9 +529,8 @@ sub add_fake_delegation {
         }
     }
 
-    Zonemaster::Engine->add_fake_delegation( $domain => \%data );
+    return Zonemaster::Engine->add_fake_delegation( $domain => \%data );
```

Comment:
Does this actually change anything? Nobody's picking up the new return value.

Also, could you add method documentation for add_fake_delegation that describes the possible set of return values and what they mean?

Reply:
You are right, this modification is a residue of a different way to fix the issue. I guess we only need to update the Engine for this issue. I guess we can reject this PR.